src/components/Button.js

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-function Button({ buttonName }) {
+function Button({ buttonName, color, wide }) {
+  if (wide) {
+    return (
+      <div className="btn-style-new">
+        <button type="button" className={color}>{buttonName}</button>
+      </div>
+    );
+  }
   return (
-    <div>
-      <button type="button">{buttonName}</button>
+    <div className="btn-division">
+      <button type="button" className={color}>{buttonName}</button>
     </div>
   );
 }
```

Comment:
You can simplify this if statement by using a ternary operator in the `className` property:
```JS
  return (
    <div className={wide ? 'btn-style-new' : 'btn-division'}>
      <button type="button" className={color}>{buttonName}</button>
    </div>
  );
```

Reply:
Thanks a lot for the feedback. I will do that